src/pages/Bookmarks/index.jsx

```diff
@@ -77,13 +77,16 @@ function Bookmarks() {
       .finally(() => setLoading(false));
   }, []);
 
-  const filteredMovies = bookmarkedMovies.filter(movie =>
-    movie.title.toLowerCase().includes(searchQuery.toLowerCase()) // Filter movies by title
+  const filteredMovies = bookmarkedMovies.filter(
+    (movie) =>
```

Comment:
The better way is

```javascript
const isMatch = !searchQuery || 
                (movie.title?.toLowerCase().includes(searchQuery.toLowerCase()) || 
                false);
```

                
Notice these 2 things.
1. the use of `?` operator.
2. every case in a new line.

Reply:
//made utility fucn.  
const filterContentBySearch = (contentList, searchQuery, key) => {
    if (!searchQuery) return contentList;
    const lowerCaseQuery = searchQuery.toLowerCase();
    return contentList.filter(
      (content) =>
        content[key] && content[key].toLowerCase().includes(lowerCaseQuery)
    );
  };